application.go

```diff
@@ -1,9 +1,18 @@
 package main
 
 import (
+	"os"
 	"project-symi-backend/app/infrastructure"
+
+	"github.com/joho/godotenv"
 )
 
 func main() {
-	infrastructure.Router.Run()
+	err := godotenv.Load()
+	print(err)
```

Comment:
Print(err) is not good. Exit looks better.
https://stackoverflow.com/questions/4278293/how-do-i-return-from-func-main-in-go

Reply:
Yup, trying the exit with code 500, will see how it goes!